scripts/win32.bat

```diff
@@ -69,78 +69,93 @@ Function GetOperatingSystemDevice()
 	Next
 End Function
 
+Function IsWMIObjectValid(WMIObject)
+	On Error Resume Next
```

Comment:
Why is this only a one-space indent, when the rest of the file seems to use two-spaces?